hack/images

```diff
@@ -52,6 +52,18 @@ if [ -n "$localmode" ]; then
   attestFlags=""
 fi
 
+outputFlag="${outputFlag},annotation.org.opencontainers.image.title=BuildKit"
+if [ -n "$GITHUB_SHA" ]; then
+  outputFlag="${outputFlag},annotation.org.opencontainers.image.revision=$GITHUB_SHA"
+fi
+if [ -n "$GITHUB_REPOSITORY" ]; then
+  outputFlag="${outputFlag},annotation.org.opencontainers.image.source=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
+  outputFlag="${outputFlag},annotation.org.opencontainers.image.url=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
+fi
+if [ -n "$versionTag" ]; then
+  outputFlag="${outputFlag},annotation.org.opencontainers.image.version=$versionTag"
+fi
```

Comment:
this should be done only on push so moved in https://github.com/moby/buildkit/pull/3554/files#diff-d131ef7e8954a6ffbaa57cfce3e105863587a19e310500d0c7c55b9f52038bfdR47

Reply:
Sounds good, thank you!

Is there any more to do, or is this now ready to be merged?

Reply:
sorry forgot to set "request for changes", we should check `localmode` is not set for this code otherwise it would break as docker exporter does not support annotations

Reply:
also add `GITHUB_SERVER_URL` before https://github.com/moby/buildkit/blob/36ffdf7e6d63983598f28c03eb3571e4b370661f/hack/util#L7

Reply:
we should also check `if [ "$GITHUB_ACTIONS" = "true" ]`

Reply:
I've updated this MR addressing your concerns. How does it look now?